compilation errors - LOG_SCOPE

I am getting the following compilation errors related to `LOG_SCOPE` with current `master`:
```
make[1]: Entering directory `/Users/boyceg/sfw/libmesh/master-objs-dbg'
  CXX      src/solvers/libmesh_dbg_la-petsc_nonlinear_solver.lo
src/solvers/petsc_nonlinear_solver.C:66:3: error: use of undeclared identifier 'LOG_SCOPE'
  LOG_SCOPE("residual()", "PetscNonlinearSolver");
  ^
src/solvers/petsc_nonlinear_solver.C:308:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("jacobian()", "PetscNonlinearSolver");
    ^
src/solvers/petsc_nonlinear_solver.C:448:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("postcheck()", "PetscNonlinearSolver");
    ^
src/solvers/petsc_nonlinear_solver.C:769:3: error: use of undeclared identifier 'LOG_SCOPE'
  LOG_SCOPE("solve()", "PetscNonlinearSolver");
  ^
4 errors generated.
make[1]: *** [src/solvers/libmesh_dbg_la-petsc_nonlinear_solver.lo] Error 1
  CXX      src/solvers/libmesh_dbg_la-tao_optimization_solver.lo
src/solvers/tao_optimization_solver.C:51:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("objective()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:99:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("gradient()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:150:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("hessian()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:207:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("equality_constraints()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:259:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("equality_constraints_jacobian()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:308:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("inequality_constraints()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:360:5: error: use of undeclared identifier 'LOG_SCOPE'
    LOG_SCOPE("inequality_constraints_jacobian()", "TaoOptimizationSolver");
    ^
src/solvers/tao_optimization_solver.C:462:3: error: use of undeclared identifier 'LOG_SCOPE'
  LOG_SCOPE("solve()", "TaoOptimizationSolver");
  ^
src/solvers/tao_optimization_solver.C:612:3: error: use of undeclared identifier 'LOG_SCOPE'
  LOG_SCOPE("get_dual_variables()", "TaoOptimizationSolver");
  ^
9 errors generated.
make[1]: *** [src/solvers/libmesh_dbg_la-tao_optimization_solver.lo] Error 1
```
I don't think I have changed configuration settings recently.

(I will try wiping out the build directory and recompiling, but may not get to it until late...)

--- a/src/solvers/petsc_nonlinear_solver.C
+++ b/src/solvers/petsc_nonlinear_solver.C
@@ -24,6 +24,7 @@
 // C++ includes
 
 // Local Includes
+#include "libmesh/libmesh_logging.h"
 #include "libmesh/nonlinear_implicit_system.h"
 #include "libmesh/petsc_nonlinear_solver.h"
 #include "libmesh/petsc_linear_solver.h"

--- a/src/solvers/tao_optimization_solver.C
+++ b/src/solvers/tao_optimization_solver.C
@@ -25,6 +25,7 @@
 // C++ includes
 
 // Local Includes
+#include "libmesh/libmesh_logging.h"
 #include "libmesh/petsc_vector.h"
 #include "libmesh/petsc_matrix.h"
 #include "libmesh/dof_map.h"

